lnbits/tasks.py

```diff
@@ -167,15 +184,22 @@ async def perform_balance_checks():
         await asyncio.sleep(60 * 60 * 6)  # every 6 hours
 
 
-async def invoice_callback_dispatcher(checking_id: str):
+async def payment_callback_dispatcher(checking_id: str):
     """
     Takes incoming payments, sets pending=False, and dispatches them to
     invoice_listeners from core and extensions.
     """
-    payment = await get_standalone_payment(checking_id, incoming=True)
+    payment = await get_standalone_payment(checking_id)
     if payment and payment.is_in:
-        logger.trace(f"sse sending invoice callback for payment {checking_id}")
+        logger.trace(f"sse sending callback for incoming payment {checking_id}")
```

Comment:
did you test this with internal payments?

Reply:
I am sorry does it mean some payments between LNBits wallets?

Reply:
Bump @callebtc

Reply:
Sorry I was a bit slow to respond. Yes, I meant internal payments between LNbits users / wallets.

Reply:
I will look into it.

Reply:
Tested it. Indeed it is not really working with internal payments now. But also it seems like there is another problem with Payments objects generally.

https://github.com/lnbits/lnbits/issues/1066

Reply:
What i see for internal wallets is that only `is_in` payment events are somehow get into `payment_callback_dispatcher`. It is not entirely clear to me how payments settled in the core because I did even run on critical failure when tried to explore one solution.

I think that outgoing payments are somehow missed inside this function. May be you have some ideas @callebtc otherwise I will try to figure out what exactly happens with regard to internal payments.